src/lib/simulation/tob.ts

```diff
@@ -96,6 +96,11 @@ export class TheatreOfBloodClass {
 		const loot = new Bank();
 		for (let i = 0; i < 3; i++) {
 			loot.add(NonUniqueTable.roll());
+
+			// OSB only: Remove dupe elite clue scrolls
+			if (loot.amount('Clue scroll (elite)') > 1) {
+				loot.remove('Clue scroll (elite)', 1);
+			}
```

Comment:
- Seems like it wont remove all extra clues, unless its guaranteed you can only get {0,1,2}. Perhaps remove `numClues - 1`
- If you want code to definitely only run in OSB, you can check `if (BOT_TYPE === 'osb')`. The upside is no need to worry about removing it in BSO when merging. You don't have to do this though.

Reply:
Not sure what you're referring to with the `numClues -1`. But I think I found a solution that should work for all scenarios, I'll add it soon.